modules/dnn/src/int8layers/quantization_utils.cpp

```diff
@@ -146,6 +225,37 @@ class QuantizeLayerImpl CV_FINAL : public QuantizeLayer
         return true;
     }
 #endif
+    void processInputOutput(std::vector<Mat>& inputs, std::vector<Mat>& outputs)
+    {
+        CV_Assert(inputs.size() == 1 || inputs.size() == 2 || inputs.size() == 3);
+
+        // Scale and zeropoint taken as input
+        if (inputs.size() > 1)
+        {
+            scalesMat = inputs[1];
+
+            copyMatToVec(scalesMat, scales);
+
+            if(scalesMat.total() > 1) is1D = true;
+
+
+            if (inputs.size() > 2)
+            {
+                zeropointsMat = inputs[2];
+                CV_Assert(zeropointsMat.total() ==  scalesMat.total());
+                copyMatToVec(zeropointsMat, zeropoints);
+            }
+
+            if (is1D)
+            {
+                MatShape inputShape = shape(inputs[0]);
+                broadcastScaleAndZeropoint(scalesMat, zeropointsMat, scales, zeropoints, inputShape, axis, block_size);
+            }
+        }
```

Comment:
Scale and zero point are just needed to be broadcast once. After that, they should not be processed again.

Reply:
The code block inside `if(inputs.size() > 1)` is executed only when scale and zero point are provided as inputs to the layer. In this case, the `broadcastScaleAndZeropoint` function in the `finalize` method is not called because the necessary data (scale and zero point) was not available at the time of the layer's creation.

Reply:
But the next inference still gets scale and zero point from inputs. The point is we avoid this from the second time.